.github/workflows/documentation-build.yml

```diff
@@ -0,0 +1,32 @@
+name: documentation-build
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  documentation_build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup TARDIS Environment 
+        uses: conda-incubator/setup-miniconda@v1 
+        with:
+            environment-file: tardis_env3.yml
```

Comment:
Try adding the `mamba-version` parameter as described [here](https://github.com/conda-incubator/setup-miniconda/#example-6-mamba). I think it's not working now, but just try.